app/controllers/applies_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AppliesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @applies = current_user.applies
+  end
+
+  def create
+    @apply = Apply.new(project_id: params[:project], user_id: current_user.id)
```

Comment:
read about strong params and use them. Makes code more readable, also rename params[:project] to params[:project_id]